types/file/content.pp

```diff
@@ -0,0 +1,2 @@
+# @summary Validate a file content attribute
+type Stdlib::File::Content = Variant[String, Sensitive[String], Deferred[String]]
```

Comment:
TIL: this can also be [Binary](https://github.com/puppetlabs/puppet/blob/main/lib/puppet/type/file/content.rb#L66)